public/keymaps/k/keebwerk_nano_slider_default.json

```diff
@@ -0,0 +1,23 @@
+{
+  "keyboard": "keebwerk/nano_slider",
+  "keymap": "default_4f1a62c",
+  "commit": "4f1a62ca1d8e1cd1528305bda6a315489738c29f",
+  "layout":"LAYOUT",
+  "layers":[
+    [
+      "TO(1)",
+      "KC_1",     "KC_2",     "KC_3",
+      "KC_4",     "KC_5",     "KC_6",     "KC_0"
+    ],
+    [
+      "TO(2)",
+      "RGB_TOG",  "RGB_MOD",  "RGB_VAI",
+      "KC_TRNS",  "RGB_RMOD", "RGB_VAD",  "KC_TRNS"
+    ],
+    [
+      "TO(0)",
+      "KC_VOLD",  "KC_VOLU",  "KC_F24",
+      "KC_MRWD",  "KC_MFFD",  "KC_F23",  "KC_MPLY"
+    ]
```

Comment:
On one hand, this doesn't match the keymap that's on qmk_firmware.

On the other, I think this one is better. :/

https://github.com/qmk/qmk_firmware/blob/cf40c33c907c44753bb145b9f2d5107447422fbc/keyboards/keebwerk/nano_slider/keymaps/default/keymap.c#L35-L48

Reply:
I agree. I'll change the one in the keyboard on my PR on that

Reply:
`!ping` Are you still planning on/intending to update this on the qmk_firmware side?